deploy/gk-deploy

```diff
@@ -20,13 +20,15 @@ LOG_FILE=''
 VERBOSE=0
 CLI=''
 GLUSTER=0
-TEMPLATES="templates"
+KUBE_TEMPLATES_DEFAULT="./kube-templates"
+OCP_TEMPLATES_DEFAULT="./ocp-templates"
+TEMPLATES="openshift"
```

Comment:
why default to openshift instead of kubernetes?

really, "openshift" does not seem correct ? should it be ocp-templates or kube-templates ?